src/lib/Containers/__stories__/MyProfile.story.tsx

```diff
@@ -1,13 +0,0 @@
-// TODO: Make this work again after Relay Modern migration
-
-// import { storiesOf } from "@storybook/react-native"
-// import React from "react"
-// import * as Relay from "react-relay/classic"
-
-// import Routes from "../../relay/routes"
-// import MyProfile from "../MyProfile"
-
-// storiesOf("My Account").add("Root", () => {
-//   const profileRoute = new Routes.MyProfile()
-//   return <Relay.RootContainer Component={MyProfile} route={profileRoute} />
-// })
```

Comment:
There's a _lot_ of these i the codebase I discovered, I swear that the GMV iOS team avoids storybooks :P